recipes/libnegf/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "libnegf" %}
+{% set version = "1.0" %}
+{% set build = 0 %}
+{% set mpi = mpi or "nompi" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a9f634a104b7e5a7f8e69a91c294f541747822d87a67d2e924d2fd502c1ce0bf
+  patches:
+    - cmake-version.patch
+
+{% if mpi == "nompi" %}
+{% set build = build + 100 %}
+{% endif %}
+build:
+  number: {{ build }}
+  skip: True  # [win]
+
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+  string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}
+
+  {% if mpi != "nompi" %}
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+  {% endif %}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - ninja
+    - fypp
+  host:
+    - {{ mpi }}  # [mpi != "nompi"]
+    - mpifx * mpi_{{ mpi }}_*  # [mpi != 'nompi']
+    - libblas
+    - liblapack
+  run:
+    - {{ mpi }}  # [mpi != "nompi"]
+    - {{ pin_compatible('mpifx') }} mpi_{{ mpi }}_*  # [mpi != 'nompi']
+    - libblas
+    - liblapack
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libnegf.dylib  # [osx]
+    - test -f $PREFIX/lib/libnegf.so  # [linux]
+
+about:
+  home: https://github.com/{{ name }}/{{ name }}
```

Comment:
The jinja proliferation like this is not really readable or nice. One cannot click on that URL and open the documentation when reading the recipe.